src/transformStateWithClones.js

```diff
@@ -8,6 +8,49 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const cloneState = [];
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
For... of... loop can make the code just a bit more readable and understandable in this case, although this is not crucial

Reply:
Agree

Reply:
`For of` will better

Reply:
Thanks for the review, it helped me a lot)